<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformaciones</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>500.000000000000000</width>
      <height>500.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QComboBox" name="FiguraCB">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione Figura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polígono</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arco</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cubo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prisma Rectangular</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Prisma Triangular</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cono</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="transformLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>321</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Selecciona la figura que deseas transformar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GradosLB_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Grados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LadosLB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Lados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GradosLB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Grados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="EscalaLB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Escala:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="GradosTB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZOOMINTB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ZOOMOUTTB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="polyEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="arcEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>60</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>500.000000000000000</width>
      <height>500.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QLineEdit" name="RadioTB">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>75</string>
    </property>
   </widget>
   <widget class="QLabel" name="Radio_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="centroX">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="centroY">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Centrolabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Centro:</string>
    </property>
   </widget>
   <widget class="QLabel" name="centroX_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="centroY_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Figura:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Figura Transformada:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="largo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="largolabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>68</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Largo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TransCB">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione Transformación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rotación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Traslación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zoom In</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zoom Out</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reflexión Vertical</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reflexión Horizontal</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Revertir_Todo">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Revertir Todo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TrasladarBTN">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Trasladar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1067</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
